Source/WebKit/UIProcess/API/Cocoa/NSAttributedString.mm

```diff
@@ -139,11 +139,14 @@ + (WKWebViewConfiguration *)configuration
     if (!configuration) {
         configuration = adoptNS([[WKWebViewConfiguration alloc] init]);
         [configuration setProcessPool:adoptNS([[WKProcessPool alloc] init]).get()];
-        [configuration setWebsiteDataStore:[WKWebsiteDataStore nonPersistentDataStore]];
+        [configuration setWebsiteDataStore:[WKWebsiteDataStore _dataStoreForNSAttributedString]];
```

Comment:
Kind of unfortunate that you have to switch *two* knobs to turn off SSO. Is there a plan to get us down to just one?

Reply:
I'm not 100% confident the second knob is critical to turn off, but marking the expected configuration as non-AppSSO seems reasonable.